packages/internal/src/build/babel/api.ts

```diff
@@ -105,12 +109,13 @@ export const getApiSideBabelPlugins = (
      *
      */
     ['@babel/plugin-transform-runtime', BABEL_PLUGIN_TRANSFORM_RUNTIME_OPTIONS],
-    // // still needed for jest.mock
-    forJest && [
+    [
       'babel-plugin-module-resolver',
       {
         alias: {
           src: './src',
+          // adds the paths from [ts|js]config.json to the module resolver
+          ...getPathsFromConfig(config.api),
```

Comment:
I acknowledge the tech debt in these configs, but instead of getting rid of the `forJest` parameter, could we mimic the logic that the web side has where if `forJest` is `true` it conditionally adds in the src alias? 

https://github.com/redwoodjs/redwood/blob/8f672d3fca0b311b68f398a937a721ca93e82a31/packages/internal/src/build/babel/web.ts#L37

Reply:
I couldn't see the two sides of that condition, one being: add the src alias when `forJest` is true, but we also want it to be there when its false. So I couldn't figure out how to do this.

Adding this would cause the tests to fail, so i decided to leave it, but still very curious to understand what you meant, im pretty sure I missed something.
```
 forJest ? rwjsPaths.api.src : './src', 
```